tasks/models.py

```diff
@@ -1,5 +1,15 @@
 from django.contrib.auth.models import User
 from django.db import models
+from enum import Enum
+
+
+class ScoreModeEnum(Enum):
+    SUM = "SUM"
+    GROUP_MIN = "GROUP_MIN"
+
+
+class DiffModeEnum(Enum):
+	C1 = "C1"
```

Comment:
Probably okay but in the future, in case we need to add more diff modes, we should give them descriptive names (like CUSTOM, INT, FLOAT, ...)